docker/test_performance_layers.sh

```diff
@@ -55,3 +55,7 @@ echo "Run is finished successfully!"
 for file in "${output_files[@]}"; do
   check_layer_log "${file}"
 done
+
+# Check that log file's contents matches what is expected.
+FileCheck-11 FileCheckTests/compile_time_patterns --input-file \
```

Comment:
We should "install" FileCheck with `update-alternatives` so that FileCheck defaults to the currently installed version. This way when we update llvm tools we won't have to also update this script and will be able to run this script outside of CI.